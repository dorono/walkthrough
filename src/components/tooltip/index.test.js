import React from 'react';
import {shallow} from 'enzyme';
import Tooltip from './index';

const defaultProps = {
    children: <h1>Test</h1>,
};

describe('Tooltip', () => {
    it('should render without errors', () => {
        shallow(<Tooltip {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<Tooltip {...defaultProps} />)).toMatchSnapshot();
    });
});
