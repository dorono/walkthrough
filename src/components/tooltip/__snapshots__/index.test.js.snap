// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip should match snapshot 1`] = `
<div
  className="tooltip pointing-up undefined"
>
  <h1>
    Test
  </h1>
</div>
`;
